operations/security_and_authentication/Change_credentials_on_ServerTech_PDUs.md

```diff
@@ -0,0 +1,133 @@
+# Change credentials on ServerTech PDUs
```

Comment:
Capitalization change: "Change Credentials on ServerTech PDUs"

Reply:
Should be resolved fixed now.